src/content/general/architcture/openstack/index.md

```diff
@@ -0,0 +1,164 @@
+
+#### Cilium
+
+Cilium is open source software for providing and transparently network connectivity in Kubernetes clusters. Cilium operates at Layer 3/4 to provide traditional networking and security services. Additionally it protects and secure applications offering different enhanced features on top. In the workload cluster is used as container network only.
+
+#### Cert exporter
+
+It is a Prometheus exporter that exposes a set of metrics regarding certificates and tokens so we can be aware in case any of those expire.
+
+#### Net exporter
+
+Net exporter is a Prometheus exporter for exposing network information in that runs as daemon set in every node and track network and DNS errors.
+
+#### Metric server
+
+Metrics Server is a component that implements the [Kubernetes Metrics API](https://kubernetes.io/docs/tasks/debug/debug-cluster/resource-metrics-pipeline/#metrics-api) to provide basic data of the container running in the cluster.
+
+#### Node exporter
+
+It is a Prometheus exporter for hardware and OS metrics exposed by *NIX kernels.
+
+## Workload segregation and account model
+
+When starting out with our platform many of our customers are at the beginning of their journey to a distributed and highly resilient micro-service architecture. This is often a radically different approach to organizing and managing computing resources. This is mostly about abstracting the complexity of cluster creation and management. It opens up new possibilities on how to isolate applications and access to the infrastructure. The two most common reasons for customers to segregate applications over different clusters and/or accounts are security and separation of concerns.
+
+Once Kubernetes arrived on the scene, the promise of having all your applications in a single cluster seemed invaluable, as container isolation, namespaces, and other Kubernetes features allow you to isolate the workloads. But as time passed several drawbacks were found that discourage this approach. Container isolation is not perfect, in the end workloads still share a kernel and some cluster components that can affect each other. Though it can be mitigated using Role-Based Access Control (RBAC), Network Policies or Pod Security Policies, it requires expertise and knowledge. At the same time, as you continue to add more and more applications into the cluster, the traffic increases. This potentially affects DNS latency, ingress handling, or the attack surface. In addition, having multiple tenants in the cluster increases the possibility to affect each other when a central component is tuned for a specific scenario. Potentially, it can significantly impact the cluster lifecycle, i.e. upgrades, due to the extra effort in communication and planning across various teams.
```

Comment:
```suggestion
Once Kubernetes arrived on the scene, the promise of having all your applications in a single cluster seemed invaluable, as container isolation, namespaces, and other Kubernetes features allow you to isolate the workloads. But as time passed several drawbacks were found that discourage this approach. Container isolation is not perfect, in the end workloads still share a kernel and some cluster components that can affect each other. Though it can be mitigated using Role-Based Access Control (RBAC), Network Policies or Pod Security standards (and the Pod Security Policies they are replacing), it requires expertise and knowledge. At the same time, as you continue to add more and more applications into the cluster, the traffic increases. This potentially affects DNS latency, ingress handling, or the attack surface. In addition, having multiple tenants in the cluster increases the possibility to affect each other when a central component is tuned for a specific scenario. Potentially, it can significantly impact the cluster lifecycle, i.e. upgrades, due to the extra effort in communication and planning across various teams.
```